src/ofdm.c

```diff
@@ -187,6 +188,7 @@ struct OFDM *ofdm_create(const struct OFDM_CONFIG *config) {
         ofdm_ntxtbits = 4;
         ofdm_ftwindowwidth = 11;
         ofdm_timing_mx_thresh = 0.30f;
+        ofdm_phase_est_bandwidth = LOW_PHASE_EST;
```

Comment:
Should be high bandwidth by default, as this is the initial state for the state machine.  The state machine then switches to low bandwidth once we get sync.

Reply:
My bad.